src/content/docs/en/recipes/dynamically-importing-images.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Dynamically Import Images
+description: Learn how to dynamically import images using Vite's import.meta.glob function
+i18nReady: true
+type: recipe
+---
+import FileTree from '~/components/FileTree.astro'
+
+When working with [images](/en/guides/images/), there will be times where you will need to dynamically import the image paths. 
+
+In this recipe, you will learn how to dynamically import your images the correct way using Vite's `import.meta.glob` function. 
+
+## Recipe
+
+1. Create a new `assets` folder under the `src` directory and add your images inside that new folder. 
+
+<FileTree>
+- src/
+  - assets/
+    - example-image.jpg
+    - example-image-2.jpg
+    - example-image-3.jpg
```

Comment:
```suggestion
    - avatar-1.jpg
    - avatar-2.png
    - avatar-3.jpeg
```
If these are going to be images associated with people, I think it's a nice touch to name them something like `avatar` or `headshot` or `person` etc. It reinforces the entire narrative around the thing you're building!

Also, since your import below is built to handle different file extensions, it could be an idea to show those off here by making each image a different file type.